release-20191212 (#25)
publiccodenet/blog: _posts/2019-12-12-notes-from-community-call-10-October-2019.md

---
title: "Notes from community call 10 October 2019"
date: 2019-12-12
author: Jan Ainali
type: blogpost

> excerpt: News from the Foundation of Public Code as of 10 October 2019.

Reviewer: ```suggestion
excerpt: Recent news roundup from the Foundation for Public Code.
```

categories:
  - Community call
---

# Notes from community call - 10 October 2019

> As a matter of transparency, we post the notes from this community call even though no one attended the call.

Reviewer: ```suggestion
For transparency, we're sharing the update we prepared for this community call even though no one attended the call.
```
